test/micro/org/openjdk/bench/java/util/concurrent/Maps.java

```diff
@@ -71,6 +76,11 @@ public void initTest() {
             key[i] = rng.next();
         }
         position = key.length / 2;
+
+        staticMap = new ConcurrentHashMap<>();
+        for (int i = 0; i < nkeys; ++i) {
+            staticMap.put(rng.next(), rng.next());
+        }
```

Comment:
Can just merge this loop with the one above, reusing `rng.next()` for both key and values for `staticMap`.